suites/message/multisig.go

```diff
@@ -220,258 +223,79 @@ func TestMultiSigActor(t *testing.T, factory state.Factories) {
-		duck, duckId := td.NewAccountActor(drivers.SECP, initialBal) // 104
-		var initialSigners = []address.Address{alice, bob}
-
-		multisigAddr := utils.NewIDAddr(t, 1+utils.IdFromAddress(duckId))
-		createRet := td.ComputeInitActorExecReturn(alice, 0, 0, multisigAddr)
-
-		td.MustCreateAndVerifyMultisigActor(0, valueSend, multisigAddr, alice,
-			&multisig_spec.ConstructorParams{
-				Signers:               initialSigners,
-				NumApprovalsThreshold: initialNumApprovals,
-				UnlockDuration:        unlockDuration,
-			},
-			types.MessageReceipt{
-				ExitCode:    exitcode_spec.Ok,
-				ReturnValue: chain.MustSerialize(&createRet),
-				GasUsed:     big_spec.NewInt(1794),
-			})
-
-		// alice fails to add a singer since this method can only be called by the multisig actors wallet address
-		td.ApplyMessageExpectReceipt(
-			td.MessageProducer.MultisigAddSigner(multisigAddr, alice, multisig_spec.AddSignerParams{
-				Signer:   chuck,
-				Increase: false,
-			}, chain.Value(big_spec.Zero()), chain.Nonce(1)),
-			types.MessageReceipt{
-				ExitCode:    exitcode_spec.ErrForbidden,
-				ReturnValue: drivers.EmptyReturnValue,
-				GasUsed:     big_spec.NewInt(1_000_000),
-			})
-
-		// success when multisig actor calls the add signer method
-		td.ApplyMessageExpectReceipt(
-			td.MessageProducer.MultisigAddSigner(multisigAddr, multisigAddr, multisig_spec.AddSignerParams{
-				Signer:   chuck,
-				Increase: false,
-			}, chain.Value(big_spec.Zero()), chain.Nonce(0)),
-			types.MessageReceipt{
-				ExitCode:    exitcode_spec.Ok,
-				ReturnValue: drivers.EmptyReturnValue,
-				GasUsed:     big_spec.NewInt(521),
-			})
-		// assert that chuck is now a signer
-		td.AssertMultisigState(multisigAddr, multisig_spec.State{
-			Signers:               append(initialSigners, chuck),
-			NumApprovalsThreshold: initialNumApprovals,
-			NextTxnID:             0,
-			InitialBalance:        valueSend,
-			StartEpoch:            td.ExeCtx.Epoch,
-			UnlockDuration:        unlockDuration,
-		})
-
-		// add another signer and increase the number of signers required
-		td.ApplyMessageExpectReceipt(
-			td.MessageProducer.MultisigAddSigner(multisigAddr, multisigAddr, multisig_spec.AddSignerParams{
-				Signer:   duck,
-				Increase: true,
-			}, chain.Value(big_spec.Zero()), chain.Nonce(1)),
-			types.MessageReceipt{
-				ExitCode:    exitcode_spec.Ok,
-				ReturnValue: drivers.EmptyReturnValue,
-				GasUsed:     big_spec.NewInt(587),
-			})
-		// assert that duck is noe a signer and the number of approvals required increased
-		td.AssertMultisigState(multisigAddr, multisig_spec.State{
-			Signers:               append(initialSigners, chuck, duck),
-			NumApprovalsThreshold: initialNumApprovals + 1,
-			NextTxnID:             0,
-			InitialBalance:        valueSend,
-			StartEpoch:            td.ExeCtx.Epoch,
-			UnlockDuration:        unlockDuration,
-		})
-
-	})
-
-	t.Run("remove signer and decreases threshold", func(t *testing.T) {
```

Comment:
The diff is weird, but I deleted tests below the "add signer" one. They are not exercising any novel VM or message execution behaviour, only the multisig actor itself. To the extent they did anything that the actor unit tests don't do, they should move there.